packages/core/src/login/oidc/IClient.ts

```diff
@@ -30,4 +30,13 @@
 export default interface IClient {
   clientId: string;
   clientSecret?: string;
+  // The client name is a human-readable name for the client application. It
```

Comment:
I don't know why `IClient` is marked as `@hidden`, but this looks like it would be helpful as a TSDoc comment?